### ISSUE TITLE
"News"-section todo

### ISSUE THREAD

**Issue by GWRon** [bmx-ng/bmx-site#22]
For now "news" leads to "blogs" which is empty too.

Why not add the various commit-logs of all the repositories the BlitzMaxNG project is consisting of?

So fetch all commits/activities and glue them together so only the date orders "the projects activities".

**Comment by GWRon** [comment]
Eg. we could use this one here:
https://github.com/kvz/on-the-githubs

**Comment by woollybah** [comment]
Your example isn't very useful. It seems general activities (like comments) aren't very helpful at all.
I accept, if there was a list of commits, on a per project basis, it might be useful.

**Comment by GWRon** [comment]
Will have to look further - but I assume there are libraries out there which just fetch what you want (from the API) and render the jscon response into some div containers.

**Comment by woollybah** [comment]
Yeah... something that takes the output of this, I suppose : https://api.github.com/repos/bmx-ng/bmk/commits
and does something useful with it...
